server/src/RelayHttpServer.go

```diff
@@ -19,7 +19,7 @@ import (
 	"time"
 )
 
-const VERSION = "0.3.5"
+const VERSION = "0.3.6"
```

Comment:
I think its more than a minor change - its a clear API change.

Reply:
This one is the the server version, no API change in it.

Reply:
done